src/binary/Yast.cc

```diff
@@ -166,6 +168,81 @@ void set_ruby_source_location(VALUE file, VALUE lineno)
   YaST::ee.setLinenumber(FIX2INT(lineno));
 }
 
+/**
+ * Is the function name an UI function which returns a symbol?
+ * @param  function_name name of the function
+ * @return true if function_name is one of "UserInput", "TimeoutUserInput"
+ *   or "PollInput"
+ */
+static bool ui_symbol_function(const char *function_name)
+{
+    return strcmp(function_name, "UserInput") == 0  ||
+        strcmp(function_name, "TimeoutUserInput") == 0 ||
+        strcmp(function_name, "PollInput") == 0;
+}
+
+/**
+ * Is the YCP value :debugHotkey symbol?
```

Comment:
hmm, I prefer here behavior, so something like `returns true if input is debug hotkey`
It is more flexible if we decide to rename it or having more symbols. Then contract of this function is to detect it and it is bug if it is not updated. Otherwise it looks like it work as expected, but in fact do not work.